packages/app/package.json

```diff
@@ -17,8 +17,8 @@
   "dependencies": {
     "@fontsource/fira-code": "^4.5.9",
     "@fontsource/inter": "^4.5.10",
-    "@fuel-ts/providers": "0.0.0-master-d5e02003",
     "@fuel-ts/contract": "0.0.0-master-d5e02003",
+    "@fuel-ts/providers": "0.0.0-master-d5e02003",
```

Comment:
Should we add package.json to the prettier rule, it looks like it is changing the order a lot

Reply:
But it's not better that it changes the order and format json as expected? 🙄

Reply:
Yeah, this is the idea, we should format always them it will start to keep the order and add only single modifications if it's already enabled, it's good.